src/config/config.prod.json

```diff
@@ -4,5 +4,6 @@
   "apiPort": 3333,
   "apiPath": "",
   "jsonWebToken": "",
-  "logErrors": false
+  "logErrors": false,
+  "sipiUrl": "https://iiif.dasch.swiss:1024/"
```

Comment:
the port is not needed here

Reply:
According to [docs](https://docs.knora.org/03-apis/api-v2/editing-values/#upload-files-to-sipi): 
```
HTTP POST to http://sipihost/upload?token=TOKEN

The first step is to upload one or more files to Sipi, using a multipart/form-data request, where sipihost represents the host and **port** on which Sipi is running:
```

I've just put protocol, host and port in one string, but this could be separated.

Reply:
It's only a detail here. The prod config will anyway be updated from server conf.

Reply:
Does it make any difference when I remove port? Should I do it so?

Reply:
Yes it makes a difference, because on the defined URL this port does not exist. But as I said, this prod-conf file will anyway not be used. It's just an example